draft-zf-community-dev-fund-2-proposal.rst

```diff
@@ -0,0 +1,486 @@
+halving), 20% of the block subsidy of each block SHALL be allocated to a "Dev 
+Fund" that consists of the following three slices:
+
+* 28% for the Bootstrap Project (denoted **BP slice**);
+* 32% for the Zcash Foundation (denoted **ZF slice**);
+* 40% for the Financial Privacy Foundation, for "Zcash Community Grants" for
+  large-scale long-term projects (denoted **ZCG slice**).
+
+Subsequently, starting at block height 3146400, 20% of the block subsidy of 
+each block SHALL be allocated to a "Dev Fund" that consists of the following 
+two slices:
+
+* 32% for the Zcash Foundation (denoted **ZF slice**);
+* 68% for the Financial Privacy Foundation, for "Zcash Community Grants" for
+  large-scale long-term projects (denoted **ZCG slice**).
+
+The slices are described in more detail below. The fund flow will be 
+implemented at the consensus-rule layer, by sending the corresponding ZEC to 
+the designated address(es) for each block. This Dev Fund will end at block 
+height 3566400 (unless extended/modified by a future ZIP).
+
+
+BP slice (Bootstrap Project)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This slice of the Dev Fund will flow as charitable contributions from the 
+Zcash Community to the Bootstrap Project, the newly formed parent organization 
+to the Electric Coin Company. The Bootstrap Project is organized for exempt 
+educational, charitable, and scientific purposes in compliance with Section 
+501(c)(3), including but not limited to furthering education, information, 
+resources, advocacy, support, community, and research relating to 
+cryptocurrency and privacy, including Zcash. This slice will be used at the 
+discretion of the Bootstrap Project for any purpose within its mandate to 
+support financial privacy and the Zcash platform as permitted under Section 
+501(c)(3). The BP slice will be treated as a charitable contribution from the 
+Community to support these educational, charitable, and scientific purposes.
+
+
+ZF slice (Zcash Foundation)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This slice of the Dev Fund will flow as charitable contributions from the 
+Zcash Community to ZF, to be used at its discretion for any purpose within its 
+mandate to support financial privacy and the Zcash platform, including: 
+development, education, supporting community communication online and via 
+events, gathering community sentiment, and awarding external grants for all of 
+the above, subject to the requirements of Section 501(c)(3). The ZF slice will 
+be treated as a charitable contribution from the Community to support these 
+educational, charitable, and scientific purposes.
+
+
+ZCG slice (Zcash Community Grants)
+~~~~~~~~~~~~~~~~~~~~~~~
+
+This slice of the Dev Fund is intended to fund independent teams entering the
+Zcash ecosystem, to perform major ongoing development (or other work) for the
+public good of the Zcash ecosystem, to the extent that such teams are 
+Available and effective.
```

Comment:
```suggestion
available and effective.
```
(If "Available" was intended to have a more specific definition, retain the capital but define it in the Terminology section, and consider capitalising other uses.)